etc/cp-ksql-server/README.md

```diff
@@ -0,0 +1,41 @@
+# kafka-data transformer (KSQLDB)
+
+Reference: https://docs.ksqldb.io/
+
+The KSQLDB Kafka data transformer is able to register Consumer/Producers to Kafka that transform data in a topic and
+publish the results to another topic.
+
+The provided KSQLDB _questionnaire_response_observations.sql_ and _questionnaire_app_events_observation.sql_ SQL files
+transform, respectively, the _questionnaire_response_ and _questionnaire_app_event_ topics and publish the data to the
+_ksql_observations_ topic. The _ksql_observations_ topic is consumed by the Kafka-JDBC-connector used for the by the
+RADAR-base Data Dashboard backend service (see: [20-data-dashboard.yaml](../../helmfile.d/20-dashboard.yaml)).
+
+When transformation of other topics is required, new SQL files can be added to this directory. These new files should be
```

Comment:
Is there a web interface for this app to add the queries? Seems to be a bit complicated to add queries with helm

Reply:
No, there is not. For now, transformation logic is added via SQL files registered in KSQLDB.

Reply:
I see